x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_policy_create_multi_step_extension.tsx

```diff
@@ -6,13 +6,110 @@
-import type { PackagePolicyCreateMultiStepExtensionComponentProps } from '@kbn/fleet-plugin/public';
+import { EuiText, EuiIcon, EuiFlexGroup, EuiFlexItem, EuiLink, EuiSpacer } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import styled from 'styled-components';
 
 /**
- * TBD: A component to be displayed in multi step onboarding process.
+ * A component to be displayed in multi step onboarding process.
  */
-export const EndpointPolicyCreateMultiStepExtension =
-  memo<PackagePolicyCreateMultiStepExtensionComponentProps>(({ newPolicy }) => {
-    return <></>;
-  });
+export const EndpointPolicyCreateMultiStepExtension = memo(() => {
+  return (
+    <Container>
+      <LargeCustomSpacer />
+      <Title />
+      <MediumCustomSpacer />
+
+      <EuiFlexGroup>
+        <EuiFlexItem grow={false} style={{ alignItems: 'center' }}>
+          <LargeSecurityLogo />
+        </EuiFlexItem>
+
+        <EuiFlexItem>
+          <div>
+            <Features />
+          </div>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+
+      <MediumCustomSpacer />
+      <Details />
+      <LargeCustomSpacer />
+    </Container>
+  );
+});
 EndpointPolicyCreateMultiStepExtension.displayName = 'EndpointPolicyCreateMultiStepExtension';
+
+const Container = styled('div')`
+  padding: 0 23px;
+`;
+
+const Title = () => (
+  <EuiText>
+    <h3>
+      <FormattedMessage
+        id="xpack.securitySolution.endpoint.policy.multiStepOnboarding.title"
+        defaultMessage="We'll save your integration with our recommended defaults."
+      />
+    </h3>
+  </EuiText>
+);
+
+const LargeCustomSpacer = () => (
+  <>
+    <EuiSpacer size="m" />
+    <EuiSpacer size="xxl" />
+  </>
+);
+
+const MediumCustomSpacer = () => (
+  <>
+    <EuiSpacer size="m" />
+    <EuiSpacer size="xl" />
+  </>
+);
+
+const LargeSecurityLogo = () => (
+  <EuiIcon type="logoSecurity" style={{ width: '128px', height: '128px' }} />
+);
+
+const Features = () => (
+  <EuiFlexGroup alignItems="center" gutterSize="s" responsive={false}>
+    <EuiFlexItem grow={false}>
+      <EuiIcon type="check" />
+    </EuiFlexItem>
+
+    <EuiFlexItem>
+      <EuiText size="m">
+        <p>
+          <FormattedMessage
+            id="xpack.securitySolution.endpoint.policy.multiStepOnboarding.feature"
+            defaultMessage="Windows, Mac, and Linux event collection"
+          />
+        </p>
+      </EuiText>
+    </EuiFlexItem>
+  </EuiFlexGroup>
+);
+
+const Details = () => (
+  <EuiText size="m" color="subdued">
+    <p>
+      <FormattedMessage
+        id="xpack.securitySolution.endpoint.policy.multiStepOnboarding.details"
+        defaultMessage="You can change this later by editing the Endpoint Security integration agent policy.
+      Read more about Endpoint security configuration in our {docsPage}."
+        values={{
+          docsPage: (
+            <EuiLink href="https://www.elastic.co/guide/en/security/master/configure-endpoint-integration-policy.html">
```

Comment:
You have to use kibana docs service here. This will show the link with the stack version. You can check how it works [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_generic_errors_list.tsx#L21) . You will also have to update kibana docs types to add the new one if it doesn't exists yet.

Reply:
[209dde3](https://github.com/elastic/kibana/pull/142395/commits/209dde3de424951c9741c6947a89af8db1b6d141)